.github/workflows/ci-cd.yml

```
@@ -323,73 +451,101 @@ jobs:
         with:
           node-version: 22
 
-      - name: Calculate and apply version bump
-        id: bump
+      - name: Calculate Release Version
+        id: version_calc
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2034:warning:45:5: CURRENT appears unused. Verify use (or export if used externally)

(shellcheck)